testing/test_mark.py

```diff
@@ -1041,7 +1041,11 @@ def assert_test_is_not_selected(keyword):
 )
 @pytest.mark.filterwarnings("ignore")
 def test_parameterset_extractfrom(argval, expected):
-    extracted = ParameterSet.extract_from(argval)
+    class DummyItem:
+        def warn(self, warning):
```

Comment:
we should either trigger it and use a p pytest.warns block, or mock it and do  call check

Reply:
Good catch, done